.github/workflows/ci.yml

```diff
@@ -139,113 +120,25 @@ jobs:
           cat .pre-commit-config.yaml
 
       - name: Install project (needed for mypy check)
-        run: poetry install -E core -E backend
+        run: |
+          set -ex
+          poetry install -E core -E backend
+        # We skip _parsec.so compilation given Mypy only cares about _parsec.pyi
+        env:
+          POETRY_LIBPARSEC_BUILD_STRATEGY: no_build
```

Comment:
Can you move that to the job scope ? 
That would make it simpler to find

Reply:
Fixed in [4e5b760](https://github.com/Scille/parsec-cloud/pull/3761/commits/4e5b7603206f48551b1165b2176c27c4d9a0b7b0)

Reply:
I prefer to have this here to make it obvious (this config file is big and it's easy to miss out an env variable)